src/features/import/utils/createPreviewData.spec.ts

```diff
@@ -0,0 +1,202 @@
+import createPreviewData from './createPreviewData';
+import { ColumnKind, Sheet } from './types';
+import { describe, it } from '@jest/globals';
+
+describe('createPreviewData()', () => {
+  describe('When it has a person index for preview', () => {
```

Comment:
As far as I can tell, this extra `describe()` adds no extra value, because there is only one.

In jest, `describe` is used to group tests. Sometimes, one might use multiple nested `describe` blocks to create "sub-groups" of tests. This affects how the tests are printed by the test runner, so that tests that belong in the same group (within the same `describe`) are grouped when printed.

In this test file, there is only one "sub-group" ("When it has a person index for preview"), so the group doesn't actually offer any additional context.

I would just remove the inner `describe` entirely.